packages/yoga/src/NavigationMenu/web/Menu/Menu.tsx

```diff
@@ -85,9 +84,11 @@ const Menu = ({ avatar: Avatar, subtitle, title, onClick }: MenuProps) => {
       {Avatar}
 
       <StyledTextContainer>
-        <StyledText>
-          {title || <Skeleton type="text" variant="body2" width="100%" />}
-        </StyledText>
+        {title ? (
+          <StyledText>{title}</StyledText>
+        ) : (
+          <Skeleton type="text" variant="body2" width="100%" />
```

Comment:
I'm in doubt. If there is no title it will show a "loading" state forever? Is this the intended behaviour? or is this skeleton "not animated"?

Reply:
After analysis with @tiagogomes772, I concluded that neither "Skeleton" nor "isLoading" state are necessary in the "Menu" component - which is the content of the "Header". The title should be rendered conditionally as it is for subtitle.

If we should need a "Loading" state, it should be a "global" skeleton that is shown while the menu is fetching data.

Reply:
As discussed offline, a loading prop was added to the `Switcher` component